ChangeLog.md

```diff
@@ -1,5 +1,121 @@
 # Changelog for haskell-language-server
 
+## 0.2
+
+- Use cabal-plan from Hackage
+([#185](https://github.com/haskell/haskell-ide-engine/pull/185) by @georgefst)
```

Comment:
Very sure this is the wrong url :)
```suggestion
([#185](https://github.com/haskell/haskell-language-server/pull/185) by @georgefst)
```

Reply:
oops, copy and paste. I checked the structure, not the detail. Thanks.